frontend/src/services/events.js

```diff
@@ -0,0 +1,42 @@
+import axios from 'axios'
+import { v4 as uuidv4 } from 'uuid';
+
+// Get request
+const get = async () => {
+    const response = await axios.get('http://localhost:2121/events')
+    return response.data;
+}
+    
+// Post request
+const create = async userData => {
+    const response = await axios.post('http://localhost:2121/events', {
+        ...userData,
+        id: uuidv4()
```

Comment:
The uuid package can be removed. It turns out json-server will generate the id for us.